frame/proxy/src/benchmarking.rs

```diff
@@ -92,13 +92,13 @@ benchmarks! {
 		// ... and "real" is the traditional caller. This is not a typo.
 		let real: T::AccountId = account("caller", 0, SEED);
 		let call: <T as Trait>::Call = frame_system::Call::<T>::remark(vec![]).into();
-		add_announcements::<T>(T::MaxPending::get(), Some(caller.clone()), None)?;
```

Comment:
Not that `proxy` extrinsic did not depend on announcements, so I have removed this unneeded line.